src/models/layerMetadata/layerDEMMetadata.ts

```diff
@@ -29,6 +29,8 @@ export interface ILayerMetadata {
   undulationModel: UndulationModel | undefined;
   dataType: DataType | undefined;
   noDataValue: NoDataValue | undefined;
+  sensorType: SensorType[] | undefined; //sensors
```

Comment:
Shouldn't it be:
sensors: string[] | undefined 
region: string[] | undefined

Reply:
we didn't won't to break other types compatibility - so we changed only our data